ofl/twinklestar/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,9 @@
+<p>
+Twinkle Star is a cute and fun juvenile script. It comes in two styles: A Script style with a more curvy quality and a casual style with more children writing resemblance.
```

Comment:
Is there still 2 styles? Which one is this one?

Reply:
He means within the font (stylistic set) but perhaps it's not clear enough. I'll change it.

Reply:
Updated now